src/app/pagination-item/pagination-item.component.html

```diff
@@ -0,0 +1,4 @@
+<li *ngIf=current class="pagination__current">
```

Comment:
We can try to combine the two `li`'s here since they're mostly the same.

```suggestion
<li [ngClass]="{'pagination__current': current}">
```

The above should set `pagination__current` as the class if `current` is `true`, see https://angular.io/api/common/NgClass.